modules/custom_operations/user_ie_extensions/tokenizer/python/openvino_tokenizers/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '2023.3.0.0'
```

Comment:
Maybe it is better to get a version from openvino and then add `.0` to it? That way we didn't need to manually set it every time.